colcon_core/argument_parser/destination_collector.py

```diff
@@ -26,16 +28,21 @@ def get_destinations(self):
         :returns: The destination names
         :rtype: list
```

Comment:
The return type was fixed in a follow up commit: 0a844f741dd83c5d6cdfd18ed2ae685427a7b62a